libcxx/docs/Status/Cxx17.rst

```diff
@@ -40,7 +40,7 @@ Paper Status
 
 .. note::
 
-   .. [#note-P0067] P0067: ``std::(to|from)_chars`` for integrals has been available since version 7.0. ``std::to_chars`` for ``float`` and ``double`` since version 14.0 ``std::to_chars`` for ``long double`` uses the implementation for ``double``.
+   .. [#note-P0067] P0067: ``std::(to|from)_chars`` for integrals has been available since version 7.0. ``std::to_chars`` for ``float`` and ``double`` since version 14.0 ``std::to_chars`` for ``long double`` uses the implementation for ``double``. ``std::from_chars`` for ``float`` and ``double`` since version 20.0.
```

Comment:
I think this comment would benefit a rewrite, to note what is actually still open. E.g.
```suggestion
   .. [#note-P0067] P0067: for integer types, ``std::(to|from)_chars`` has been available since version 7.0; for ``float`` and ``double``, ``std::to_chars`` since version 14.0 and ``std::from_chars`` since version 20.1. Support is complete except for ``long double``, which currently uses the implementation for ``double``.
```

Reply:
I left a comment before, but forgot to publish it :-/
`to_chars` has `float`, `double`, and `long double`, where `long double` is `double`
`from_chars` only has `float` and `double` support, and no `long double` support at all.
(Adding proper `long double` support is on my todo list and using LLVM-libc that will be easier, however a lot of work will be to write proper tests for `long double`. So I feel your suggestion is not correct.

Feel free to propose different wording if you have a better suggestion.